src/core/style/mixins.scss

```diff
@@ -125,15 +125,10 @@
   }
 }
 
-@mixin rtl-prop($ltr-prop, $rtl-prop, $value) {
+@mixin rtl-prop($ltr-prop, $rtl-prop, $value, $reset-value) {
```

Comment:
The point of the PR isn't to reduce the CSS as much as possible, but rather to remove the redundancies while staying as close to the old functionality as possible.